Source/Tools/LinuxEmulation/VDSO_Emulation.cpp

```diff
@@ -431,10 +480,22 @@ void LoadHostVDSO() {
     x64::Handler_getcpu = x64::VDSO::getcpu;
     x32::Handler_getcpu = x32::VDSO::getcpu;
   }
+
+
+  SymbolPtr = dlsym(vdso, "__kernel_getrandom");
+  if (!SymbolPtr) {
+    SymbolPtr = dlsym(vdso, "__vdso_getrandom");
+  }
+
+  if (SymbolPtr) {
+    VDSOHandlers::GetRandomPtr = reinterpret_cast<VDSOHandlers::GetRandomType>(SymbolPtr);
+    x64::Handler_getrandom = x64::VDSO::getrandom;
+    ///< 32-bit doesn't have getrandom vdso
```

Comment:
Why does this comment start with `///<`?

Reply:
Removed the weird